packages/components/src/components/buttons/Button/Button.stories.tsx

```
@@ -76,12 +76,13 @@ export const Button: StoryObj<ButtonProps> = {
         },
         icon: {
-            options: {
-                'No icon': null,
-                ...variables.ICONS.reduce((acc, icon) => ({ ...acc, [icon]: icon }), {}),
-            },
+            options: [null, ...variables.ICONS],
             control: {
                 type: 'select',
+                labels: {
+                    'No icon': null,
+                    ...variables.ICONS.reduce((acc, icon) => ({ ...acc, [icon]: icon }), {}),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)